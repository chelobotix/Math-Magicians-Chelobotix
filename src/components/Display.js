const Display = () => (
  <div className="display">
    <p className="display">0</p>
  </div>
);

export default Display;
