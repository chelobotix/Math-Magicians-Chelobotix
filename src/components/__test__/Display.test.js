import renderer from 'react-test-renderer';
import Display from '../Display';

describe('Display Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<Display calcObj={{}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
