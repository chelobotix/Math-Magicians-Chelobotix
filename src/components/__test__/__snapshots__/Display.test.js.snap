// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component Tree 1`] = `
<div
  className="display"
>
  <p
    className="display"
  />
</div>
`;
